tests/coverage/test_gcovr.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -eu
+# for info see 'read.me'
+
+# return the canonicalized absolute pathname
+realpath() {
+  OURPWD=$PWD
+  cd "$(dirname "$1")"
+  LINK=$(readlink "$(basename "$1")")
+  while [ "$LINK" ]; do
+    cd "$(dirname "$LINK")"
+    LINK=$(readlink "$(basename "$1")")
+  done
+  REALPATH="$PWD/$(basename "$1")"
+  cd "$OURPWD"
+  echo "$REALPATH"
+}
+
+DIR_SRC=$(dirname "$(realpath $0)")
+DATE_TIME=$(date +"%y-%m-%d-%H%M%S")
+DIR_REPORT=$(realpath "gcovr/report_gcovr_${DATE_TIME}")
+DIR_CI=$(realpath "../ci/travis")
+DIR_GTEST=$(realpath "$DIR_CI/googletest")
+GCOVR_REPORT="report.html"
+CTEST_REPORT="ctest.log"
+NUMBER_THREADS="-j 4"
+
+mkdir -p $DIR_REPORT
+# Installing gtest
+if [ ! -d $DIR_GTEST ]
+  then
+    pushd $DIR_CI
+    # Setup test
+    . setup_gtest.sh
+    popd
+fi
+export CMAKE_SYSTEM_PREFIX_PATH="$DIR_GTEST/install"
+# Build facelift with flags
+cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_FLAGS="-fprofile-arcs -ftest-coverage" -DFACELIFT_BUILD_TESTS=ON -DCMAKE_SYSTEM_PREFIX_PATH=$CMAKE_SYSTEM_PREFIX_PATH .. && make
```

Comment:
To configure AND build in one command, do:
cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_FLAGS="-fprofile-arcs -ftest-coverage" -DFACELIFT_BUILD_TESTS=ON -DCMAKE_SYSTEM_PREFIX_PATH=$CMAKE_SYSTEM_PREFIX_PATH --build .. -j

Reply:
Ok